### ISSUE TITLE
datatable without grouping

### ISSUE THREAD

**Issue by markballenger** [dc-js/dc.js#1164]
I'd like to be able to use a dc datatable without specifying a group. 

This workaround doesn't work for me:
http://stackoverflow.com/questions/26803161/dc-datatable-without-grouping-the-rows 

I am trying to initialize a DataTables from DataTables.js postRender as a means to quickly/easily leverage pagination/sorting/searching. It mostly works, but because the DOM structure has appended several tbody tags, it is getting in the way of the DataTables.js pagination/filtering. 

I can fix the DOM my self during post render for now, but it seems like this should be a feature OOB.

**Comment by XaserAcheron** [comment]
If you're using the 2.0 betas, there's a function to toggle this now -- `myChart.showGroups(false)` will do the trick.

**Comment by gordonwoodhull** [comment]
@markballenger, does showGroups not work for your use case?

**Comment by markballenger** [comment]
No, the Dom created multiple body tags still. Messes with datatables

**Comment by gordonwoodhull** [comment]
I see. While multiple `tbody` is apparently valid html, DataTables probably doesn't know what to do with that. dc shouldn't need those tags but that's the hierarchical way it's building the table right now. Yuk.

**Comment by markballenger** [comment]
Yeah, it works if I fix the Dom post redraw. It's just ugly to have to do
that.

**Comment by MiroBabic** [comment]
@markballenger  can you paste here your solution pls? I have similar issue with multiple tbody tags and not working datatables because of it
